geode-redis/src/integrationTest/java/org/apache/geode/redis/general/ShutdownIntegrationTest.java

```diff
@@ -56,6 +57,7 @@ public void shutdownActuallyShutsDownTheServer() {
   }
 
   @Test
+  @Ignore("will be remedied in: https://github.com/apache/geode/pull/5204")
```

Comment:
It would probably be better for you just to wait for: https://github.com/apache/geode/pull/5209. Otherwise this ignore will need to be removed in yet another pr.

Reply:
gotcha

Reply:
Merged it in!